README.rst

```diff
@@ -0,0 +1,36 @@
+core-workflow
+=============
+Issue tracker and relevant tools for CPython's workflow
+
+.. image:: https://travis-ci.org/python/core-workflow.svg?branch=master
+    :target: https://travis-ci.org/python/core-workflow
+
+cherry_picker 🐍 🍒 ⛏
+-------------------------------
```

Comment:
It doesn't matter; reST only cares that the dashes are *at least* as long the file; this makes sure that I don't have to care if any of those emojis are counted as more than a single letter by reST.